File: course-content/examples/switch_expressions1.java
public class switch_expressions1 {
    public static void main(String[] args) {
	
	int day = 4;
	String message = switch (day) {
	  case 1  -> "Pazartesi";
	  case 2  -> "Salı";
	  case 3  -> "Çarşamba";
	  case 4  -> "Perşembe";
	  case 5  -> "Cuma";
	  case 6  -> "Cumartesi";
	  case 7  -> "Pazar";
	  default -> throw new IllegalStateException("Invalid day: " + day);
	}; // gerekli
	System.out.println(message);

    }
}

File: course-content/examples/switch_expressions1.cs
public class switch_expressions1 {
    public static void Main(string[] args) {
        
        int day = 4;
        string message = day switch {
            1 => "Pazartesi",
            2 => "Salı",
            3 => "Çarşamba",
            4 => "Perşembe",
            5 => "Cuma",
            6 => "Cumartesi",
            7 => "Pazar",
            _ => throw new ArgumentException($"Invalid day: {day}")
        };
        Console.WriteLine(message);
    }
}